README.md

```diff
@@ -130,10 +108,40 @@ smooth-release --changelog --gh-release-all # first time using smooth-release on
     inSyncWithRemote: true,
     noUncommittedChanges: true,
     noUntrackedFiles: true
+  },
+  tasks: {
+    validations: true,
+    'npm-publish': null,
+    'npm-version': null,
+    'gh-release': null,
+    'gh-release-all': false,
+    changelog: null
   }
 }
 ```
 
-If you want to change parts of it you can define a JSON config file in the root directory of your project named `.smooth-releaserc`.
+setting a task to `null` means that `smooth-release` will prompt you every time before running the task:
+![image](https://cloud.githubusercontent.com/assets/4029499/21606902/e78f23d0-d1b2-11e6-9c17-b4bccf853856.png)
+
+If you want to change parts of the default config you can define a JSON file in the root directory of your project named `.smooth-releaserc`.
 
 The file will be recursively merged into `defaultConfig` (NB: arrays are replaced, not merged!).
+
+
+## CLI arguments
```

Comment:
Can we specify in the Readme that by using any CLI argument, all prompts are disabled?